.ci/.jenkins_nodejs.yml

```diff
@@ -0,0 +1,11 @@
+NODEJS_VERSION:
+  - "12"
+  - "12.0"
```

Comment:
While I guess it's technically going to give us more coverage, we don't currently test the http client on the `.0` versions of Node.js. So feel free to remove those from this file

Reply:
I think the *.0 versions are mostly just an advantage with testing instrumentation is applied at the right place, but _may_ sometimes help catching use of new language features added mid-way through a version. 🤔

Reply:
I've just found I resolved this particular conversation, I removed those `.0` versions, so happy to put it back if it adds any value in this particular case

Reply:
I just mentioned it because we didn't already do this, so if y'all were trying to replicate our existing Travis setup, then this wasn't needed. And if we want to be economical about how many resources we spin up, then this was a place where I think we could save a little without worrying too much about it. But if we want to add all the `x.0` versions I'm of course fine with that as well.